src/modding/ModManager.cs

```diff
@@ -522,30 +763,81 @@ private void UpdateSelectedModInfo()
     {
         if (selectedMod != null)
         {
+            selectedModInfoBox.Visible = true;
+
             selectedModName.Text = selectedMod.Info.Name;
             selectedModIcon.Texture = LoadModIcon(selectedMod);
             selectedModAuthor.Text = selectedMod.Info.Author;
             selectedModVersion.Text = selectedMod.Info.Version;
-            selectedModRecommendedThriveVersion.Text = selectedMod.Info.RecommendedThriveVersion;
-            selectedModMinimumThriveVersion.Text = selectedMod.Info.MinimumThriveVersion;
-            selectedModDescription.Text = selectedMod.Info.Description;
+            selectedFromWorkshop.Text = selectedMod.Workshop ?
+                TranslationServer.Translate("THIS_IS_WORKSHOP_MOD") :
+                TranslationServer.Translate("THIS_IS_LOCAL_MOD");
+
+            if (!(string.IsNullOrEmpty(selectedMod.Info.RecommendedThriveVersion) &&
+                    string.IsNullOrEmpty(selectedMod.Info.MinimumThriveVersion)))
```

Comment:
I would not have this check here as these are mandatory fields. Okay maybe one of these was optional, but then the other should definitely be still shown.